Integrations/SlashNextPhishingIR/SlashNextPhishingIR.py

```diff
@@ -0,0 +1,976 @@
+    Execute SlashNext's URL/reputation API against the requested URL reputation command with the given parameters
+    @:return: None
+    """
+    args = demisto.args()
+
+    if 'url' not in args:
+        return_error('Missing required command argument "url", Please provide "url" and retry!')
+
+    response = invoke_snx_api(api_type='url-repute', url=args.get('url'))
+
+    if response.get('errorNo') != 0:
+        return_error('API Returned, "{}:{}"!'.format(response.get('errorNo'), response.get('errorMsg')))
+    else:
+        url_data = response.get('urlData')
+        snx_ioc_cont, dbot_score_cont, url_cont = get_snx_url_ioc_context(url_data)
+
+        ec = {
+            'SlashNext.IoC(val.Value == obj.Value)': snx_ioc_cont[0],
+            'DBotScore': dbot_score_cont,
+            'URL': url_cont
+        }
+
+        title = 'SlashNext Phishing Incident Response - URL Reputation\n'\
+                '##### url = {}'.format(url_data.get('url'))
+
+        if response.get('normalizeData').get('normalizeStatus') == 1:
+            title = title + ' *\n*' + response.get('normalizeData').get('normalizeMessage')
+
+        md = tableToMarkdown(
+            title,
+            snx_ioc_cont,
+            ['Value',
+             'Type',
+             'Verdict',
+             'ThreatStatus',
+             'ThreatName',
+             'ThreatType',
+             'FirstSeen',
+             'LastSeen']
+        )
+
+        publish_md_and_ec(md, snx_ioc_cont, ec)
+
+
+def url_scan_command():
+    """
+    Execute SlashNext's URL/scan API against the requested URL scan command with the given parameters
+    @:return: None
+    """
+    args = demisto.args()
+
+    if 'url' not in args:
+        return_error('Missing required command argument "url", Please provide "url" and retry!')
+
+    response = invoke_snx_api(api_type='url-scan', url=args.get('url'))
+
+    if response.get('errorNo') == 1:
+        demisto.results(
+            'Your Url Scan request is submitted to the cloud and may take up-to 60 seconds to complete.\n'
+            'Please check back later using "snx-scan-report" command with Scan ID = {} or '
+            'running the same "snx-url-scan" command one more time.'.format(response.get('urlData').get('scanId')))
+    elif response.get('errorNo') != 0:
+        return_error('API Returned, "{}:{}"!'.format(response.get('errorNo'), response.get('errorMsg')))
+    else:
+        url_data = response.get('urlData')
+        scanid = url_data.get('scanId')
+        snx_ioc_cont, dbot_score_cont, url_cont = get_snx_url_ioc_context(url_data, is_scan=True)
+
+        ec = {
+            'SlashNext.IoC(val.Value == obj.Value)': snx_ioc_cont[0],
+            'DBotScore': dbot_score_cont,
+            'URL': url_cont
+        }
+
+        title = 'SlashNext Phishing Incident Response - URL Scan\n'\
+                '##### url = {}'.format(url_data.get('url'))
+
+        if response.get('normalizeData').get('normalizeStatus') == 1:
```

Comment:
```suggestion
        if response.get('normalizeData', {}).get('normalizeStatus') == 1:
```

Reply:
Mandatory field

Reply:
Can you guarantee this convention will stay as is and backward compatibility will be kept forever?